mne_bids/write.py

```diff
@@ -511,6 +511,27 @@ def _participants_tsv(raw, subject_id, fname, overwrite=False):
         }
     )
 
+    # Make sure that all entries to data are lists that
+    # contain scalars (i.e. not further lists). Fix if possible
+    for key in data.keys():
+        cur_value = data[key]
```

Comment:
```suggestion
    for key, cur_value in data.items():
```

Reply:
The issue here is that I do:

```python
data[key] = new_value
```

within the loop. And afaik, one should never change what is iterated over. It might be save here, because care is taken that only the current entry is changed... But I think, it is safer this way....